### PR TITLE
New package: taizen-0.1.0

### LINKED ISSUE #1105: Package Request: Taizen
https://github.com/NerdyPepper/taizen

New Rust based terminal browser for wikipedia.

### PR DESCRIPTION
**Pull request by travankor**
closes #1105

needs a license and tagged release versioning before merging.

### CODE REVIEW

**Comment by NerdyPepper** [comment]
Taizen is now under the MIT license. You may go ahead with the merge.

**Comment by travankor** [comment]
I don't have merge access to the repo, but we will have to see if the githash is ok.

File: srcpkgs/taizen/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'taizen'
+pkgname=taizen
+version=0.1.0
+revision=1
+_githash=d86da2ce6c687401fe8e6e34fc69cd22aaba72e1
```
Comment (cr6git) [inline srcpkgs/taizen/template]: you use a git commit dated Aug 5, 2018.
`version=0.0.0.20180805`

File: srcpkgs/taizen/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+wrksrc=${pkgname}-${_githash}
```
Comment (cr6git) [inline srcpkgs/taizen/template]: quotes.
`wrksrc="${pkgname}-${_githash}"`

File: srcpkgs/taizen/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/taizen/template]: see also https://github.com/voidlinux/void-packages/pull/6707#discussion_r121899848

**Comment by travankor** [comment]
fixed.

(would be nice if xlint checked for quotes)

File: srcpkgs/taizen/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'taizen'
+pkgname=taizen
+version=0.0.0.20180808
+revision=1
+_githash=39f77b5c1128d710b12244e433677daaf87c085e
+wrksrc="${pkgname}-${_githash}"
+hostmakedepends="cargo libressl-devel pkg-config ncurses-devel"
+depends="ncurses"
```
Comment (cr6git) [inline srcpkgs/taizen/template]:
````
hostmakedepends="cargo pkg-config"
makedepends="libressl-devel ncurses-devel"
````
And `depends="ncurses"` seems unnecessary.

File: srcpkgs/taizen/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+short_desc="Browse mediawiki pages from the command line"
+maintainer="travankor <travankor@tuta.io>"
+license="MIT"
+homepage="https://github.com/NerdyPepper/taizen"
+distfiles="${homepage}/archive/${_githash}.tar.gz"
+checksum=7de71509bc2b4494af1e103ddb18e0eeafe0bd90c797d6cae343c44f583bfd15
+nocross=yes
```
Comment (cr6git) [inline srcpkgs/taizen/template]: You could explain how the cross-compilation fails.

**Comment by travankor** [comment]
all rust packages are marked no cross in Void.

someone needs to work on the rust cross toolchain (I do not know the actual reason why rust is all nocross)?

**Pull request merged**